src/@next/Combobox/components/OptionList/OptionListStyle.ts

```diff
@@ -5,11 +5,14 @@ import { space4, space8 } from '../../../utilities/spacing';
 
 interface OptionListContainerProps {
   textInputWidth?: number;
+  height?: string;
 }
 
 export const OptionListContainer = styled.div<OptionListContainerProps>`
   width: ${props => props.textInputWidth}px;
+  height: ${props => props.height};
   padding: ${space8} 0;
+  overflow-y: auto;
```

Comment:
hmmm I think Popover has scrollable style, just that I forgot to include it, can we manage that there? Or else will  need to do for every popoever children 🤔 
![image](https://user-images.githubusercontent.com/105256293/224879397-0b795103-b48a-4868-8e4a-2b1d634f01fd.png)

Reply:
@marthayaputra I see! thanks! let me check that 👀

Reply:
@marthayaputra I've applied your suggestion